charter.html

```diff
@@ -377,6 +377,13 @@ <h2 id=process>Process</h2>
 <a href=https://www.w3.org/community/about/agreements/cla/>W3C Community
 Contributor License Agreement (CLA)</a>.
 
+<p>The Community Group will not work on or publish a <a
+href="https://www.w3.org/community/reports/">specification</a> unless the
+maintainers of at least two independent interoperable implementations have
```

Comment:
how about "unless at least two organizations have volunteered to provide implementation experience"?